entity-framework/core/querying/built-in-functions.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Built-in Functions - EF Core
```

Comment:
FYI, I'm compiling a list of SQLite translations in the following format.

.NET | SQL | Added in
--- | --- | ---
dateTime.AddMilliseconds(value) | `datetime(@dateTime, (@value / 1000.0) \|\| ' seconds')` | EF Core 2.2